web/themes/interledger/css/navigation.css

```diff
@@ -190,7 +190,8 @@
   .site-links-wrapper {
     grid-area: 1 / 1 / 2 / 4;
     display: grid;
-    grid-template-columns: minmax(10em, 1fr) 1fr 1fr;
+    /* the 123px below: ~ the width of the language switcher, so that menu is centered */
+    grid-template-columns: 10em 1fr calc(10em - 123px);
```

Comment:
Let's avoid "magic numbers". It's always better practice to put a value in a variable and name it. Then when people look at it in the future they can see what this value represents. 

I tried explicitly setting the width on the language switcher to 123px and it breaks. Keep in mind the width value changes as the screen size changes, it causes the text size to change since our fonts are set to be smoothly responsive.